core/lib/wallet_creator/src/main.rs

```diff
@@ -27,26 +32,26 @@ fn eth_random_account_credentials() -> (H160, H256) {
 }
 
 async fn create_new_wallet() -> Result<Wallet<PrivateKeySigner, RpcProvider>, anyhow::Error>{
-    let (eth_address, eth_private_key) = eth_random_account_credentials();
-    println!("ETH Address: {:?}",eth_address);
-    println!("Private key: {:?}",eth_private_key);
+    let (main_eth_address, main_eth_private_key) = eth_main_account_credentials();
```

Comment:
@kenny08gt Would be possible to receive the credentials from command line? Ideally i think it would be great to create a random account only if no address/private key has been received from command line. WDYT?

Reply:
Just added it